lib/runtimes/ServerlessNodeRuntime.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
Again, sorry to nitpick, but can you rename these to `ServerlessRuntime[Type]` so that they are grouped together in our lib folder?